Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt

```diff
@@ -23,6 +23,8 @@ class IncidenceCard(parent: ViewGroup) :
         item: StatisticsCardItem,
         payloads: List<Any>
     ) -> Unit = { item, payloads ->
-        item.stats as IncidenceStats
+        val stats = item.stats as IncidenceStats
+        val value = stats.keyFigures[0].value
```

Comment:
I'm not sure if this is safe enough. Maybe try something like
`stats.keyFigures.firstOrNull { it.rank == KeyFigureCardOuterClass.KeyFigure.Rank.PRIMARY }?.value`